code/modules/religion/honorbound/honorbound_trauma.dm

```diff
@@ -45,46 +48,53 @@
 	if(!isliving(clickingon))
 		return
 
-	var/mob/living/clickedmob = clickingon
+	var/mob/living/clicked_mob = clickingon
 	var/obj/item/weapon = honorbound.get_active_held_item()
 
-	if(!honorbound.DirectAccess(clickedmob) && !isgun(weapon))
+	if(!honorbound.DirectAccess(clicked_mob) && !isgun(weapon))
 		return
 	if(weapon?.item_flags & NOBLUDGEON)
 		return
-	if(!honorbound.combat_mode && (HAS_TRAIT(clickedmob, TRAIT_ALLOWED_HONORBOUND_ATTACK) || ((!weapon || !weapon.force) && !LAZYACCESS(modifiers, RIGHT_CLICK))))
+	if(!honorbound.combat_mode && (HAS_TRAIT(clicked_mob, TRAIT_ALLOWED_HONORBOUND_ATTACK) || ((!weapon || !weapon.force) && !LAZYACCESS(modifiers, RIGHT_CLICK))))
 		return
-	if(!is_honorable(honorbound, clickedmob))
+	check_visible_guilt(clicked_mob)
+	if(!is_honorable(honorbound, clicked_mob))
 		return (COMSIG_MOB_CANCEL_CLICKON)
 
+/// Checks a mob for any obvious signs of evil, and applies a guilty reason for each.
+/datum/brain_trauma/special/honorbound/proc/check_visible_guilt(var/mob/living/attacked_mob)
```

Comment:
```suggestion
/datum/brain_trauma/special/honorbound/proc/check_visible_guilt(mob/living/attacked_mob)
```
This is whats causing your linter issues

Reply:
Yeah I keep telling myself to nab that in the next commit and then forgetting because it's an uncommon mistake, THIS time I got it.